res/values/strings.xml

```diff
@@ -1439,68 +1426,68 @@
     <string name="preferences__mmsc_password">MMSC Password</string>
     <string name="preferences__sms_delivery_reports">SMS delivery reports</string>
     <string name="preferences__request_a_delivery_report_for_each_sms_message_you_send">Request a delivery report for each SMS message you send</string>
-    <string name="preferences__automatically_delete_older_messages_once_a_conversation_exceeds_a_specified_length">Automatically delete older messages once a conversation exceeds a specified length</string>
+    <string name="preferences__automatically_delete_older_messages_once_a_conversation_exceeds_a_specified_length">Automatically delete old messages once a conversation exceeds a specified number of messages. Use this to keep the app from taking up too much storage space, or to prevent old messages from lingering on your phone.</string>
     <string name="preferences__delete_old_messages">Delete old messages</string>
     <string name="preferences__chats">Chats and media</string>
     <string name="preferences__storage">Storage</string>
     <string name="preferences__conversation_length_limit">Conversation length limit</string>
-    <string name="preferences__trim_all_conversations_now">Trim all conversations now</string>
-    <string name="preferences__scan_through_all_conversations_and_enforce_conversation_length_limits">Scan through all conversations and enforce conversation length limits</string>
+    <string name="preferences__trim_all_conversations_now">Delete all old messages now</string>
```

Comment:
Delete is not the same as trim. Not sure about the current implementation in Signal for trimming messages...

Reply:
In Signal it is the same thing. Trimming the conversations means deleting messages older than a certain date. Those messages are erased permanently.

'"Delete all old messages" is factually correct, and is much more clear to users.

The description text underneath "Delete all old messages now" clearly indicates that  old messages" means messages older than the user specified time limit.

`<string name="preferences__scan_through_all_conversations_and_enforce_conversation_length_limits">Scan through all conversations and enforce the conversation length limit right now</string>`